FoxHound.Client/src/Comments/Comments.tsx

```diff
@@ -1,26 +1,27 @@
 import React from "react";
-import Comment from './Comment';
+import Comment from "./Comment";
 
 export interface CommentModel {
-    postId : number;
```

Comment:
I just removed postId from the CommentModel. The other changes in this file are just formatting from Prettier.